### DISCUSSION TITLE
Invalid format of DKIM record.

### DISCUSSION THREAD

**Discussion by dtrckd** [postalserver/postal#2223] [Help with using Postal]
On https://toolbox.googleapps.com/apps/checkmx/

The site returns an error when testing a postal server.

![image](https://user-images.githubusercontent.com/8218696/200031451-f09a9671-951e-4b77-aad2-702e90eea2af.png)

Adding the tag `k=rsa` remove this error (as suggested in https://stackoverflow.com/questions/70515354/how-to-resolve-googles-invalid-format-of-dkim-record-error-message), but then the dkim record is no more valid on postal mail-tester.com.

Do you have any insight about this `k` tag ?

I started digging this to reduce the likelihood to my email to be marked as spam by Gmail.

**Comment by willpower232** [comment]
To be honest, I would refer to mail-tester.com over the google thing. The google one presumes you're using them for all your email so is quite biased.

mxtoolbox also has one that works well https://mxtoolbox.com/dkim.aspx

mail-tester.com will give you any spam-related hints so should be your first to satisfy.

**Reply by dtrckd** [reply]
Hi, thanks :)

yes, this is what I though also, it is when you use their smtp servers...

I am 10/10 and all green on these testers. I can't figure out why my email go to spam on gmail, quite frustrating :/

note: according to mxtoolbox, my IP is not blacklisted (except for UCEPROTECTL3 ....)

**Reply by willpower232** [reply]
Unfortunately it sounds like you've already done all you can so either theres a specific content problem or maybe your IP address (or neighbouring IP addresses) are not well reputed
